src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -397,7 +397,13 @@ private function createBlock($messages, $type = null, $style = null, $prefix = '
                 $message = OutputFormatter::escape($message);
             }
 
-            $lines = array_merge($lines, explode(PHP_EOL, wordwrap($message, $this->lineLength - $prefixLength - $indentLength, PHP_EOL, true)));
+            $length = $this->lineLength;
+
+            if (false !== (bool) preg_match('/(<\/comment>)/', $message, $match)) {
```

Comment:
`if (preg_match(...))` is way enough and consistent with the code base, no need for extra convolutions
BUT, here, a simple strpos check would be better IMHO